packages/core/src/runtime/compileAtomicCSSRule.test.ts

```diff
@@ -160,10 +207,10 @@ describe('compileAtomicCSSRule', () => {
           value: 'red',
         }),
       ).toMatchInlineSnapshot(`
-      Array [
-        ".fui-FluentProvider .foo .focus:hover{color:red;}",
-      ]
-    `);
+              Array [
+                ".fui-FluentProvider .foo .focus:hover{color:red;}",
+              ]
+          `);
```

Comment:
Revert this?

Reply:
This was applied with the formatter, I applied `yarn nx format`and check passed